project-base/config/services_test.yml

```diff
@@ -86,3 +89,67 @@ services:
     Shopsys\FrameworkBundle\Model\Pricing\PriceConverter: ~
 
     Shopsys\FrameworkBundle\Model\Pricing\Rounding: ~
+
+    Shopsys\FrameworkBundle\Model\Feed\FeedFacade: ~
+
+    Shopsys\FrameworkBundle\Component\Router\Security\RouteCsrfProtector: ~
+
+    Shopsys\FrameworkBundle\Component\Router\FriendlyUrl\FriendlyUrlFacade: ~
+
+    Shopsys\FrameworkBundle\Form\Admin\AdvancedSearch\AdvancedSearchOperatorTranslation: ~
+
+    Shopsys\FrameworkBundle\Model\Customer\User\CustomerUserFacade: ~
+
+    Shopsys\FrameworkBundle\Model\Category\CategoryVisibilityRepository: ~
+
+    Shopsys\FrameworkBundle\Model\Product\Pricing\ProductPriceRecalculator: ~
+
+    Shopsys\FrameworkBundle\Component\Setting\Setting: ~
+
+    Shopsys\FrameworkBundle\Model\Pricing\InputPriceRecalculationScheduler: ~
+
+    Shopsys\FrameworkBundle\Model\Pricing\PricingSetting: ~
+
+    Shopsys\FrameworkBundle\Model\Product\Availability\ProductAvailabilityRecalculator: ~
+
+    Shopsys\FrameworkBundle\Model\Product\Parameter\ParameterFacade: ~
+
+    Shopsys\FrameworkBundle\Model\Product\Pricing\ProductPriceRecalculationScheduler: ~
+
+    shopsys.component.elasticsearch.tracer:
+        class: Psr\Log\NullLogger
+
+    Elasticsearch\ClientBuilder:
+        class: Elasticsearch\ClientBuilder
+        factory: [Elasticsearch\ClientBuilder, create]
+        calls:
+            -   method: setHosts
+                arguments:
+                    - ['%elasticsearch_host%']
+            -   method: setTracer
+                arguments:
+                    - '@shopsys.component.elasticsearch.tracer'
+
+    Elasticsearch\Client:
+        class: Elasticsearch\Client
+        factory: ['@Elasticsearch\ClientBuilder', build]
+
+    Shopsys\FrameworkBundle\Model\Security\Authenticator:
+        arguments:
+            - '@security.token_storage'
+            - '@Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher'
+
+    Shopsys\FrameworkBundle\Component\Image\ImageLocator:
+        arguments: ['%shopsys.image_dir%']
+
+    Shopsys\ReadModelBundle\Image\ImageViewFacade: ~
+
+    Shopsys\FrameworkBundle\Component\Image\ImageFacade:
+        arguments: ['%shopsys.image_url_prefix%']
+
+    Shopsys\FrameworkBundle\Component\Router\CurrentDomainRouter:
+        tags:
+            - { name: router, priority: 70 }
+
+    Symfony\Bridge\Twig\Extension\HttpKernelRuntime:
+        arguments: ['@fragment.handler']
```

Comment:
was it necessary to copy-paste so much of services? it seems like it will be a pain to keep it in sync with services.yml file

Reply:
This is the same as above. I am lost in autowiring now, if someone else looked at it, I would love to talk about it and edit it.

Reply:
You can see commit a18654b1b420ccf089c486b948bd63a4b3e2c1fa and some newer